external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -59,7 +59,7 @@
       <Version>2.5.0</Version>
     </PackageReference>
     <PackageReference Include="System.IO.Compression.TestData">
-      <Version>1.0.4-prerelease</Version>
+      <Version>1.0.5-prerelease</Version>
```

Comment:
Yeah. The TestData package doesn't need to be changed very often so it's not too much of a hassle.